views/appView.js

```diff
@@ -0,0 +1,36 @@
+var AppView = Backbone.View.extend({
+  initialize: function(model){
+    this.model = model;
+    //console.log("Here is my appModel");
```

Comment:
I'd remove these comments in the future.  Try to treat these evals like a real project, where you'd want to not let these into the repo.